core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay

```diff
@@ -1,22 +0,0 @@
-registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-dispatcher quay.io/openshift-knative/knative-eventing-kafka-broker-dispatcher
```

Comment:
Why are these image mirroring files dropped?

Reply:
I don't know. I just re-ran the script and they're being dropped.

Gonna check again.